Common/rover_system.hpp

```diff
@@ -6,6 +6,10 @@ namespace sjsu::common
 class RoverSystem
 {
  public:
+  struct MissionControlData{
+    int is_operational;
+    int heartbeat_count;
+  };
```

Comment:
@kammce Is this possible? This is to resolve SJSURoboticsTeam/urc-central-2021/issues/194. I've not dealt with a base class that has a struct within it that is inheritable...

Reply:
This should not be in this change. A new branch and PR should be made for this. But yes, to answer your question, this is very much possible.

Reply:
See this: https://godbolt.org/z/Wb7z5KGMj